<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotRightsDlg</class>
 <widget class="QDialog" name="PlotRightsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Document rights</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="cbUseRights">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>Restrict rights</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="tbPlus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
         <property name="icon">
          <iconset resource="resource1.qrc">
           <normaloff>:/some/ico/ico/plus.png</normaloff>:/some/ico/ico/plus.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbMinus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="icon">
          <iconset resource="resource1.qrc">
           <normaloff>:/some/ico/ico/minus.png</normaloff>:/some/ico/ico/minus.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>250</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>User</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>A</string>
        </property>
        <property name="toolTip">
         <string>User can change rights</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>User properties...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlotRightsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
